pyaedt/emit_core/results/revision.py

```diff
@@ -732,3 +732,25 @@ def set_emi_category_filter_enabled(self, category: EmiCategoryFilter, enabled:
             raise RuntimeError("This function is only supported in AEDT version 2024 R1 and later.")
         engine = self.emit_project._emit_api.get_engine()
         engine.set_emi_category_filter_enabled(category, enabled)
+
+    @property
+    def license_session(self):
+        ""License session object.
```

Comment:
```suggestion
        """License session object.
```
One quote is missing.

Reply:
Thanks Alberto- fixed.